tendermint-rs/tests/integration.rs

```diff
@@ -22,6 +22,12 @@ mod rpc {
         assert_eq!(&abci_info.data, "GaiaApp");
     }
 
+    /// `/abci_query` endpoint
+    #[test]
+    fn abci_query() {
+        // TODO(tarcieri): write integration test for this endpoint
```

Comment:
Unlike the other endpoints it wasn't clear how to immediately test this in an integration test (with `gaiad`)